Warsztaty_Cz1.py

```diff
@@ -134,34 +137,135 @@
 liczba_2 = 10
 
 # 1. Wypisz drugie imię z listy 'lista_imion'
+print('\n- - - - - Zadanie 1')
+print(lista_imion[1])
 
 # 2. Zmień imię Zygmunt na Ilona
+print('\n- - - - - Zadanie 2')
+lista_imion[-1] = 'Ilona'
+print(lista_imion)
 
 # 3. Dodaj nowe imię 'Pawel' do listy (na koniec listy)
+print('\n- - - - - Zadanie 3')
+lista_imion.append('Pawel')
+print(lista_imion)
 
 # 4. Dodaj nowe imię do list 'Agata' jako drugie
+print('\n- - - - - Zadanie 4')
+lista_imion.insert(1,'Agata')
+print(lista_imion)
 
 # 5. Usuń z listy imię Jadwiga
+print('\n- - - - - Zadanie 5')
+lista_imion.remove('Jadwiga')
+print(lista_imion)
 
 # 6. Wypisz ilość imion w liście (uzywając do tego odpowiedniej metody)
+print('\n- - - - - Zadanie 6')
+print('Ilosc imion w tablicy: ',len(lista_imion))
 
 # 7. Pomnóz dwie zmienne: liczba_1 oraz liczba_2 i wypisz
+print('\n- - - - - Zadanie 7')
+print('Liczba_1: ',liczba_1)
+print('Liczba_2: ',liczba_2)
+print('Iloczyn: ',liczba_1*liczba_2)
 
 # 8. Sprawdź uzywając warunków if/else która zmienna: liczba_1 oraz liczba_2 jest większa i wypisz odpowiedni tekst na ekranie
+print('\n- - - - - Zadanie 8')
+if liczba_1 > liczba_2:
+  print('Liczba_1 jest wieksza od liczba_2')
+else:
+  print('Liczba_2 jest wieksza od liczba_1')
 
 # 9. Sprawdź czy na liście imion znajduje się imię 'Jan' i wypisz odpowiedni komunikat
+print('\n- - - - - Zadanie 9')
+if lista_imion[0] == 'Jan':
```

Comment:
a by się stało jakbym np usunął coś z tablicy albo dodał?  Aby sprawdzić czy element znajduje się na liście możesz wykorzystać taką składnię:
tablica = ["Filip", "Damian"]

if "Filip" in tablica:
    print("Jest filip")
else:
    print("nie ma filipa")